src/api/config/environments/development.rb

```diff
@@ -59,12 +59,3 @@
 
 CONFIG['frontend_host'] = "localhost"
 CONFIG['frontend_protocol'] = "http"
-
-require 'socket'
-fname = "#{Rails.root}/config/environments/development.#{Socket.gethostname}.rb"
-if File.exist? fname
-  STDERR.puts "Using local environment #{fname}"
-  eval File.read(fname)
-else
-  STDERR.puts "Custom development.#{Socket.gethostname}.rb not found - using defaults"
-end
```

Comment:
I would prefer it at least. The other option would probably be an include. Let's talk tomorrow:-)